repo-issues/lib/plugin.js

```diff
@@ -13,7 +13,7 @@ const configuration = () => {
             template: {
                 showIssues: false,
                 showPRs: true,
-                token : "Github Token"
+                token : plugin.createStringField('github token'),
```

Comment:
I've realised it wasn't reading the config properly

Reply:
💯 

eventually we'll have dateField, multiple choice, apiSecret etc and this will theme the admin config page